BUILDING.md

```diff
@@ -40,8 +40,8 @@ Support is divided into three tiers:
 | Windows      | Tier 1       | >= Windows 7 or >= Windows2008R2 | x86, x64             |                  |
 | SmartOS      | Tier 2       | >= 15 < 16.4                     | x86, x64             | see note1        |
 | FreeBSD      | Tier 2       | >= 10                            | x64                  |                  |
-| GNU/Linux    | Tier 2       | kernel >= 3.13.0, glibc >= 2.19  | ppc64le              |                  |
-| AIX          | Tier 2       | >= 7.1 TL04                      | ppc64be              |                  |
+| GNU/Linux    | Tier 2       | kernel >= 3.13.0, glibc >= 2.19  | ppc64le, >=power8    |                  |
+| AIX          | Tier 2       | >= 7.1 TL04                      | ppc64be, >=power7    |                  |
```

Comment:
Nit: can you remove the commas, elsewhere they seem to be used as or (`x64, arm` = x64 or arm, `ppc64le, >=power8` means ppc64le and power8)?

`ppc64le, >=power8` -> `ppc64le >=power8`
`ppc64be, >=power7`-> `ppc64be >=power7`